docker/1.6.0/py2/Dockerfile.gpu

```diff
@@ -1,7 +1,5 @@
-# Note: We do not need install nccl or cudnn, which were already installed in runtime container.
-# cuda-libraries and cu-blas is already installed in runtime container
-# The tag for this base image is: nvidia/cuda:10.1-cudnn7-runtime-ubuntu16.04
-FROM nvidia/cuda@sha256:b98156ec626001de2168e28a667d044e1d642ce2db7fad25e89a81977f862267
+# Note: We need to install NCCL, cuDNN, and CUDA libraries since we are using base container.
+FROM nvidia/cuda:10.1-base-ubuntu16.04
```

Comment:
Just want to verify we have signoff to use a different base image?

Reply:
Using CUDA base image plus installing some specific CUDA dev lib is working right now. Using CUDA runtime lib plus installing those CUDA dev lib would make overall image size large in comparison to using CUDA base image.